contracts/test/DaypassPaymaster.t.sol

```diff
@@ -40,60 +45,66 @@ contract DaypassPaymasterTest is Test {
 
         ownerKey = vm.envUint("OWNER_PRIVATE_KEY");
         owner = vm.addr(ownerKey);
+
         // Give some eth to the paymaster
         vm.selectFork(goerliFork);
-
         vm.deal(owner, 1001 ether);
-
-        // Paymaster stakes
+        entrypoint = EntryPoint(payable(0x0576a174D229E3cFA37253523E645A78A0C91B57));
 
         vm.startPrank(owner);
-        nftPass = new Daypass("", "", false);
+        setupHelper = new SetupHelper();
```

Comment:
Figured I'd bust out the setup helper since we were basically doing the same thing in here